core/src/services/routing.js

```diff
@@ -623,7 +622,26 @@ class RoutingClass {
     window.open(updatedExternalLink.url, updatedExternalLink.sameWindow ? '_self' : '_blank').focus();
   }
 
+  getGeneratedWCId(navNode) {
```

Comment:
Can you add some JSDoc comments for the function header here. The wc part is quite uncommented. would be nice to have some here